INSTALLING.rst

```diff
@@ -296,13 +295,54 @@ These options control CUDA compilation via ``nvcc``:
 Build the documentation
 -----------------------
 
+Requirements
+
+- Sphinx (>= 4.0 preferred)
+- sphinx_rtd_theme
+- nbsphinx
+- IPython / Jupyter notebooks
+
+Install these packages using a package manager.
+This can be the same virtual environment as used for building the package.
+Here is an example creating of a virtual environment using Python's ``venv`` and ``pip`` from the
+base HOOMD directory:
+
+.. code-block:: shell
+
+    $ python3 -m venv ~/envs/hoomd-docs-env
+    $ . ~/envs/hoomd-docs-env/bin/activate
+    (hoomd-docs-env) $ python3 -m pip -r doc-requirements.txt
+
+With conda:
+
+.. code-block:: shell
+
+    $ conda create -n hoomd-docs-env python=3.9
+    $ conda activate hoomd-docs-env
+    (hoomd-docs-env) $ conda install -c conda-forge sphinx sphinx_rtd_theme nbsphinx jupyter
+
 Use sphinx to build the documentation::
 
     $ cd /path/to/hoomd-blue
     $ cd sphinx-doc
     $ make html
 
-Then, open the documentation in ``_build/html/index.html``. On some platforms, you can use this
-command to do open the file::
+Then, open the documentation in ``_build/html/index.html``. On some platforms, namely MacOS, you can
+use this command to do open the file::
 
     $ open _build/html/index.html
+
+For Linux,::
+
+    $ xdg-open _build/html/index.html
+
+should work.
+
+To open in a browser type in the address bar,::
+
+    file:///path/to/sphinx-doc/_build/html/index.html
+
+**Common Pitfalls**
+
+- If when building there are errors regarding tutorials, submodules are either not installed or not
+  updated and must be before documentation can build without error.
```

Comment:
This is redundant with the note further up in this document. I understand that some readers may want to just build the docs, but we could make a shorter reminder of that here:
```suggestion
.. note::
    The documentation build process requires git submodules. Either clone with the
    ``--recursive`` option, or execute ``git submodule update --init`` to fetch the submodules.
```